src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/nfs/nfs-export.e2e-spec.ts

```diff
@@ -0,0 +1,87 @@
+import { ServicesPageHelper } from 'cypress/integration/cluster/services.po';
+import { BucketsPageHelper } from 'cypress/integration/rgw/buckets.po';
+
+import { NFSPageHelper } from './nfs-export.po';
+
+describe('nfsExport page', () => {
+  const nfsExport = new NFSPageHelper();
+  const services = new ServicesPageHelper();
+  const buckets = new BucketsPageHelper();
+  const bucket_name = 'e2ebucket';
+  const fsPseudo = '/fspseudo';
+  const rgwPseudo = '/rgwpseudo';
+  const editpseudo = '/editpseudo';
+  const backends = ['CephFS', 'Object Gateway'];
+  const squash = 'no_root_squash';
+  const client: object = { addresses: '192.168.0.10' };
+
+  beforeEach(() => {
+    cy.login();
+    Cypress.Cookies.preserveOnce('token');
+    nfsExport.navigateTo();
+  });
+
+  describe('breadcrumb test', () => {
+    it('should open and show breadcrumb', () => {
+      nfsExport.expectBreadcrumbText('NFS');
+    });
+  });
+
+  describe('Create, edit and delete', () => {
+    it('should create an NFS cluster', () => {
+      services.navigateTo('create');
+
+      services.addService('nfs');
+      cy.wait(15000);
```

Comment:
I was implementing a `checkServiceStatus` so that instead of just blindly waiting for 15sec and assuming that the service is created. Something like this would be better here, don't you think?  @avanthakkar https://github.com/ceph/ceph/pull/42821/files#diff-d1e362d10f8e69aea396acf9ba210a467b956eb5652838a48f53fa605b5abf75R66-R75

Reply:
Sure, if it's working fine for any service type creation, +1 from me!